src/einsteinpy/radiative_transfer.py

```diff
@@ -0,0 +1,56 @@
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+from einsteinpy.coordinates import SphericalConversion
+from einsteinpy.integrators import RK45
+
+'''create a class photon which would include a position vector and a velocity vector'''
+class radiative_transfer:
```

Comment:
Class names should follow this: https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Right Ill be careful of that from now on.